client/syncmsgs_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_test
+
+import (
+	"testing"
+
+	"perun.network/go-perun/channel"
+	"perun.network/go-perun/channel/test"
+	"perun.network/go-perun/client"
+	peruniotest "perun.network/go-perun/wire/perunio/test"
+	pkgtest "polycry.pt/poly-go/test"
+)
+
+func TestChannelSyncSerialization(t *testing.T) {
```

Comment:
You could add a short comment what this test does.

Reply:
I think the name of the test already explains that this tests the serialization of channel sync message. I have followed the same naming conventions, as that is used for other Serialization tests in `proposamsgs_test.go` and `updatemsgs_test.go`.